README.md

```diff
@@ -21,18 +21,20 @@ cd plenario
 pip install -r requirements.txt
 ```
 
-Create a PostgreSQL database for Plenario. (If you aren't
-  already running [PostgreSQL](http://www.postgresql.org/), we recommend
-  installing version 9.3 or later.)
+Create a PostgreSQL database for Plenario. (If you aren't already running
+[PostgreSQL](http://www.postgresql.org/), we recommend installing version 9.3 or
+later.) The following command creates the default database, `plenario_test`.
+This corresponds with the `DB_NAME` setting in your `plenario/settings.py` file
+and can be modified.
 
 ```
-createdb plenario_dev
+createdb plenario_test
```

Comment:
Nice. The default DB name was consistent under DataMade, but I got sloppy and introduced the difference. Good catch.